test/integrations/destinations/google_adwords_remarketing_lists/processor/data.ts

```diff
@@ -11955,7 +11955,7 @@ export const data = [
   },
   {
     name: 'google_adwords_remarketing_lists',
-    description: 'Test 20',
+    description: 'Test 20 : consent field values are discarded if it does not match allowed values',
```

Comment:
both tests are `Test 20` is that required now?

Reply:
it is just the numbering. As all of the others still have numbering. I added description after that.